master/security/namespace-policy.md

```diff
@@ -60,7 +63,31 @@ spec:
       ports:
       - 6379
 ```
+
+#### Use Kubernetes RBAC to control namespace label assignment
+
+Network policies can be applied to endpoints using selectors that match labels on either the endpoint itself, the endpoint's namespace, or the endpoint's namespace. By specifying selectors based on the endpoint's namespace we can employ Kubernetes RBAC to control which users can assign labels to namespaces. That group may be separate from the group of users who are allowed to deploy pods.
+
+In the following example, users in the **development** environment will only be allowed to communicate with other pods that have namespace labels `environment == "development"`.
```

Comment:
In the following example, users in the development environment are allowed to communicate only with other pods that have namespace labels environment == "development".